packages/slice-machine/lib/models/common/widgets/Color/Mock/index.js

```diff
@@ -2,8 +2,17 @@ import { createDefaultHandleMockContentFunction } from "../../../../../utils";
 
 export const initialValues = null;
 
+function getRandomColor() {
+  const letters = "0123456789abcdef";
```

Comment:
I would just define these outside of the function.
Between full random and generated value based on an array it seems to be the same.

Reply:
why not :)